swift-blackjack-viewsTests/DeckSpec.swift

```diff
@@ -0,0 +1,48 @@
+//  DeckSpec.swift
+
+import Quick
+import Nimble
+@testable import swift_blackjack_views
+
+class DeckSpec: QuickSpec {
+    override func spec() {
+        describe("Deck") {
+            
+            var deck : Deck = Deck()
+            
+            beforeEach({ () -> () in
+                deck = Deck()
+            })
+            
+            describe("initializer") {
+                it("should set up the deck with 52 undealt cards") {
+                    expect(deck.description).to(contain("Cards Remaining: 52"))
+                    expect(deck.description).to(contain("Cards Dealt: 0"))
+                }
+            }
+            
+            describe("drawCard") {
+                it("should return the King of Clubs from a fresh deck") {
```

Comment:
king of clubs would be the last card in the deck, so to abstract it: if you deal from the "top" (or back) of a physical deck that you just opened new, you should always draw the king of clubs.